transformations/neural_question_paraphraser/transformation.py

```diff
@@ -0,0 +1,77 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import numpy as np
+import torch
+from transformers import T5ForConditionalGeneration, T5Tokenizer
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+
+
+class NeuralParaphaserPerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
+    keywords = ["transformer-based", "highly-meaning-preserving", "high-precision"]
```

Comment:
I would not really put "high-precision" and "highly-meaning-preserving" which would mean to say that it performs close to a rule-based model. This model does produce erroneous outputs sometimes as is evident from some of the test cases even. So "high-coverage" and "high-generations" would make more sense, wouldn't it?

Reply:
yes i do agree :) thanks :)